tests/claimWin.test.ts

```diff
@@ -7,7 +7,63 @@ import { expect } from "chai";
 import { program } from "./utils/constants";
 import { IdlEvents } from "@coral-xyz/anchor";
 
-describe("Wins claiming", () => {
+describe("Wins claiming and WinClaimed Event", () => {
+  it("should emit WinClaimed event when the user claims the win", async () => {
+    const title = "Will SOL price hit $500 by 2025?1";
+    const imageUrl = "https://example.com/image.png";
+    const description = "Bet on whether SOL price will hit $500 by 2025.";
+    const optionTitle = "Yes";
+    const adminWallet = await getLocalAccount();
+    const user = await generateKeypair();
+
+    try {
+      const { poolAccountKey } = await createPool(
+        title,
+        adminWallet,
+        imageUrl,
+        description,
+      );
+      const { optionAccountKey } = await createOption(
+        optionTitle,
+        adminWallet,
+        poolAccountKey,
+      );
+      const { entryAccountKey } = await enterPool(
+        poolAccountKey,
+        optionAccountKey,
+        user,
+        new BN(1_000),
+      );
+
+      await pausePool(true, poolAccountKey, adminWallet);
+      await setWinningOption(poolAccountKey, optionAccountKey, adminWallet);
+
+      let listener: ReturnType<(typeof program)["addEventListener"]>;
+      const winClaimedListenerPromise = new Promise<
+        IdlEvents<typeof program.idl>["winClaimed"]
+      >((resolve) => {
+        listener = program.addEventListener("winClaimed", (event) => {
+          resolve(event);
+        });
+      });
+
+      await claimWin(poolAccountKey, optionAccountKey, entryAccountKey, user);
+
+      const event = await winClaimedListenerPromise;
+      await program.removeEventListener(listener);
+
+      expect(event.entry.toString()).to.eq(entryAccountKey.toString());
+      expect(event.winner.toString()).to.eq(user.publicKey.toString());
+      expect(event.pool.toString()).to.eq(poolAccountKey.toString());
```

Comment:
Test works well but instead of creating a new test can we just edit an existing test? (The successful one) to include the event testing

Reply:
Added to the last one test.